sdk/examples/wallet/accounts.rs

```diff
@@ -78,7 +77,7 @@ async fn main() -> Result<()> {
             println!("Timeout: waiting for funds took too long");
             return Ok(());
         };
-        let now = Instant::now();
+        let now = tokio::time::Instant::now();
```

Comment:
It was not the tokio one

Reply:
I know ...but probably better to use the tokio one when we already using `tokio::time`.

Reply:
Ok, was just checking in